sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Plugin.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonar.updatecenter.common;
 
+import org.apache.commons.lang.IllegalClassException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.commons.lang.IllegalClassException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)